plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -8073,7 +8073,7 @@ public void testOptimizeWithWriterScaling()
 
         Session writerScalingSession = Session.builder(optimizeEnabledSession())
                 .setSystemProperty("scale_writers", "true")
-                .setSystemProperty("writer_scaling_min_data_processed", "100GB")
```

Comment:
Should we set this to a small number instead ?
Is it guaranteed that even with the small amount of data in this test, writer scaling would have happened and some assertion would have failed here before writer scaling was disabled for OPTIMIZE ?
Is it possible to assert for something specific in the plan as that would make it more obvious that no scaling will happen ?